src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java

```diff
@@ -185,7 +185,7 @@ public void verifyNoResultsCaption() {
   public void verifyThumbnails(int number) {
     Assertion.assertNumber(thumbnails.size(), number, "checking number of thumbnails");
```

Comment:
I believe we want to assert number of thumbnails here - either those taken from the articles or placeholders. Do we need check files extensions as well?